docs/docsite/rst/community/committer_guidelines.rst

```diff
@@ -4,50 +4,58 @@
 
 Our workflow on GitHub
 ======================
 
-As a committer, you may already know this, but our workflow forms a lot of our team policies. Please ensure you're aware of the following workflow steps:
+As a committer, you may already know this, but our workflow forms a lot of our team policies. Please ensure you are aware of the following workflow steps:
 
 * Fork the repository upon which you want to do some work to your own personal repository
 * Work on the specific branch upon which you need to commit
-* Create a Pull Request back to the Ansible repository and tag the people you would like to review; assign someone as the primary "owner" of your request
-* Adjust code as necessary based on the Comments provided
-* Ask someone on the Core Team to do a final review and merge
+* Create a pull request back to the upstream repository and tag the people you would like to review; assign someone as the primary "owner" of your pull request
+* Adjust code as necessary based on the comments provided
+* Ask someone from the repository committers to do a final review and merge
 
 Addendum to workflow for committers:
 ------------------------------------
 
-The Core Team is aware that this can be a difficult process at times. Sometimes, the team breaks the rules by making direct commits or merging their own PRs. This section is a set of guidelines. If you're changing a comma in a doc, or making a very minor change, you can use your best judgement. This is another trust thing. The process is critical for any major change, but for little things or getting something done quickly, use your best judgement and make sure people on the team are aware of your work.
+The Core Team is aware that this can be a difficult process at times. Sometimes, the team breaks the rules by making direct commits or merging their own pull requests. This section is a set of guidelines. If you are changing a comma in a documentation, or making a very minor change, you can use your best judgement. This is another trust thing. The process is critical for any major change, but for little things or getting something done quickly, use your best judgement and make sure people on the team are aware of your work.
 
 Roles on Core
 =============
-* Core committers: Fine to do PRs for most things, but we should have a timebox. Hanging PRs may merge on the judgement of these devs.
-* :ref:`Module maintainers <maintainers>`: Module maintainers own specific modules and have indirect commit access through the current module PR mechanisms.
+* Core committers: Fine to do pull requests for most things, but we should have a timebox. Hanging pull requests may merge on the judgement of these developers.
+* :ref:`Module maintainers <maintainers>`: Module maintainers own specific modules and have indirect commit access through the current module pull request mechanisms.
```

Comment:
Should this be collection maintainers now? I'm not actually sure how we handle collection maintainers - my guess is the collection makes its own rules?
```suggestion
* :ref:`Collection maintainers <maintainers>`: Collection maintainers own specific collections and have commit access to them. Each collection can set its own rules for contributions.
```

Reply:
I can see community folks in BOTMETA.yml. @gundalow do you have any info if they still can `shipit` for automerge?

Reply:
Maintainers listed in BOTMETA.yml can use `shipit` to merge via the bot.

Reply:
@gundalow thanks for clarifying!

So, according to what @gundalow wrote, @acozine this is still relevant for ansible-core. I'll add what you're suggesting a bit below without removing the part about the module maintainers.

Reply:
Done